scripts/zones/West_Sarutabaruta/mobs/Numbing_Norman.lua

```diff
@@ -28,10 +28,10 @@ function onAdditionalEffect(mob,target,damage)
 end;
 
 -----------------------------------
--- onMobDeath
+-- onMobDespawn
 -----------------------------------
 
-function onMobDeath(mob, player, isKiller)
+function onMobDespawn(mob)
     checkRegime(player,mob,61,2);
```

Comment:
Well it just randomizes the respawn time multiple times instead of once,
right? Which should be fixed of course, but it's not exploitable